<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QListWidget" name="ViewTreeA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214);</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ViewTreeB">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>311</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddFolderA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/svgs/svg/folder.svg</normaloff>:/svgs/svg/folder.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>69</width>
     <height>69</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="AddFileA">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/svgs/svg/file.svg</normaloff>:/svgs/svg/file.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>69</width>
     <height>65</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="DeletA">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/svgs/svg/delete.svg</normaloff>:/svgs/svg/delete.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>69</width>
     <height>65</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="AddFolderB">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>430</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/svgs/svg/folder.svg</normaloff>:/svgs/svg/folder.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>69</width>
     <height>69</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="AddFileB">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/svgs/svg/file.svg</normaloff>:/svgs/svg/file.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>69</width>
     <height>65</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="DeletB">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/svgs/svg/delete.svg</normaloff>:/svgs/svg/delete.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>69</width>
     <height>65</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CopytoB">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/svgs/svg/left2right.svg</normaloff>:/svgs/svg/left2right.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>69</width>
     <height>35</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CopyToA">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/svgs/svg/left2right.svg</normaloff>:/svgs/svg/left2right.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>69</width>
     <height>35</height>
    </size>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
